.drone.yml

```diff
@@ -32,8 +32,10 @@ steps:
       VALIDATE_KUBERNETES_KUBEVAL: "false"
       # We use a lot of copy paste in our code, is intended. So it important to don't use this linter
       VALIDATE_JSCPD: "false"
-      VALIDATE_YAML: "false"
```

Comment:
watch out here, we would be deleting again the YAML validation

Reply:
No, by removing that line I am enabling it.

Reply:
Indeed, if you look at the CI logs, you'll see some warnings about YAML formatting.

Reply:
lol, you are right 🤦 sorry